README.md

```diff
@@ -1,4 +1,49 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql/README.md) In-progress
-2. [Core Java Apps](./core_java/README.md) In-progress
-3. [Springboot Trading REST API](./springboot/README.md) In-progress
+# James Li . Jarvis Consulting
+
+If I had to describe myself succinctly, I would say that I am a detail-oriented, responsible, and committed engineer, with a get-it-done, on-time, and as best as possible spirit. I am team-oriented, teachable, and a recent graduate from the University of Toronto with a Bachelor's degree in Computer Science. I am also a skilled Java Developer with experience creating Java application and I have the ability to implement and manage SQL database operations and data structures. On a more personal note, I enjoy traveling and playing outdoor sports such as golfing and snowboarding. In summary, I am looking for an opportunity to use my experiences and skills within the data engineering industry.
+
+## Skills
+
+**Proficient:** Java, Bash, SQL, Agile/Scrum, Git, Junit, MVC Design, REST APIs
+
+**Competent:** Docker, Springboot, Hadoop, Spark/Scala, Multithreading, Mockito, Google Cloud Platform
+
+**Familiar:** C, C++, Javascript, HTML, CSS, Salesforce
+
+## Development Projects
+
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_james](https://github.com/jarviscanada/jarvis_data_eng_james)
+
+
+- **[Cluster Monitor](./linux_sql)**: Implemented a Cluster Monitoring Solution to collect the hardware specifications of each node and monitor node resource usages. The data will be collected and stored in an RDBMS database.
+- **[Core Java Apps](./core_java)**: This project consists of three-part
+    - Developed a text search application. The application can search for a text pattern recursively in a given directory, and output matched lines to a file.
+    - Implemented is a data processing application which allows user to connect to the PostgreSQL database and perform actions to the tables.
+    - Designed a command-line application that allows users to Create, Read, and Delete Tweets. It connects to Twitter REST API using the authentication tool OAuth and performs part of CRUD implementation.
+- **[SpringBoot App](./springboot)**: Programmed a REST API designed for online stock trading simulation. The application allows a trader to sell and buy stock.
+- **[Hadoop](./hadoop)**: Implemented a data analysis project performed using Spark through Zeppelin Notebook. The Hadoop cluster was hosted on Google Cloud.
```

Comment:
project description is too short.. talk more about design pattern, libraries, testing, etc..